### PR TITLE
Add end date to check appeal page

### PR DESCRIPTION
**Pull request by mreis-ch**
[BI-8959](https://companieshouse.atlassian.net/browse/BI-8959)

Add illnessEndDate const in the prepareViewModelFromSession function of CheckYourAppealController.
Add section to check-your-appeal.njk to display the illnessEndDate if it is not null
Updated unit test descriptions and added new test to check that illnessEndDate is returned

### Work checklist

- [x] Tests added where applicable
- [x] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/controllers/CheckYourAppealController.ts
Comment (CHdlewis) [inline src/controllers/CheckYourAppealController.ts:1]: Try and keep code looking like this as when its compressed its harder to read.  Unless @mouhajer-ch has another opinion?

**Review by CHdlewis — CHANGES_REQUESTED** [review]
Looks good but add spacing into the variable names and were good to go :-)

**Review by YiannisPapastergiou — DISMISSED** [review]
All good, checked this repo and other repos for Dean's comment on spaces and he is actually right, we do use spaces after and before the brackets in node repos as a practice in CH

File: src/controllers/CheckYourAppealController.ts
Comment (mreis-ch) [reply src/controllers/CheckYourAppealController.ts:1]: Will do. I think it was when the imports got reordered, it also formatted!

Comment (mouhajer-ch) [reply src/controllers/CheckYourAppealController.ts:1]: I agree @CHdlewis, compressed makes the code harder to read. 
Something to remember, we could add [space-in-brackets](https://eslint.org/docs/rules/space-in-brackets) lint rule from eslint, when we migrate from the deprecated TSLint.

File: src/controllers/CheckYourAppealController.ts
Comment (mouhajer-ch) [inline src/controllers/CheckYourAppealController.ts:65]: It is not needed the checking "?" and will improve coverage

**Review by mouhajer-ch — APPROVED** [review]

**Comment by YiannisPapastergiou** [comment]
Since @CHdlewis is OOO and his review was solved I believe you can dismiss his review to unblock the merging @mreis-ch

**Pull request merged**
